ale/formatters/formatter.py

```diff
@@ -26,10 +26,10 @@ def to_isd(driver):
 
```

Comment:
Should the docstring be updated to say something like the ISIS _and_ CSM compatible meta-data now, since I think it is now compatible with both?

Reply:
I think keeping it generic "makes and ISD" is good enough since ISIS vs CSM compatibility isn't gonna be a thing in the long term. The separate formatted only exists as a stop-gap.